### DISCUSSION TITLE
R CMD Checks

### DISCUSSION THREAD

**Discussion by mhesselbarth** [iiasa/BNRTools#4] [General]
In my experience, we make our lives easier and the code better if we avoid any Notes, Warnings, and Errors from the R CMD Check right from the beginning. So please make sure your pushes don't result in any issues and if so, try to fix them. This will probably happen quite a bit, at least for me :)

**Comment by Martin-Jung** [comment]
Fully agree. I tried to start with some basic tests, but haven't added Github actions yet. 
We can make a general ruleset before making any PRs into `dev`: 

- Add a test for any new function that was added.
- Run locally a full check and build the package.

**Comment by mhesselbarth** [comment]
Simple CI should be on `dev` due to #2
